src/main/java/uk/gov/pay/api/model/PaymentEvent.java

```diff
@@ -1,64 +1,39 @@
 package uk.gov.pay.api.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.JsonNode;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
-import uk.gov.pay.api.model.links.PaymentEventLink;
 
-@ApiModel(value="PaymentEvent", description = "A List of Payment Events information")
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class PaymentEvent {
-    @JsonProperty("payment_id")
-    private final String paymentId;
+    @JsonProperty("charge_id")
+    private String chargeId;
```

Comment:
connector event doesn't have a chargeId

Reply:
addressed in https://github.com/alphagov/pay-publicapi/pull/633/commits/14311e18b8df8018761b8be7218dbe73e6332284